api/app/signals/apps/signals/migrations/0133_signaluser_HISTORY.py

```diff
@@ -0,0 +1,172 @@
+from django.db import migrations
```

Comment:
Can you make just 1 migration file? Combining the both into 1 please, we already have way too many migrations files and an item on our backlog to cleanup these files where possible.

Reply:
as discussed, leave this one separate. Its a huge file